### PR TITLE
[FSTORE-836] Show "Reading data with..." message and remove Hive warning

### PR DESCRIPTION
**Pull request by tdoehmen**
This PR:
- adds an animated printout "Reading data with FlyingDuck..."/ "Reading data with Hive..." in python-only client
- prints "Finished ... (0.2s)" with runtime when finished
- removed printed query and warnings when querying with hive.

JIRA Issue: FSTORE-836

Priority for Review: -

Related PRs: -

**How Has This Been Tested?**

- [ ] Unit Tests
- [ ] Integration Tests
- [x] Manual Tests on VM

**Checklist For The Assigned Reviewer:**

```
- [ ] Checked if merge conflicts with master exist
- [ ] Checked if stylechecks for Java and Python pass
- [ ] Checked if all docstrings were added and/or updated appropriately
- [ ] Ran spellcheck on docstring
- [ ] Checked if guides & concepts need to be updated
- [ ] Checked if naming conventions for parameters and variables were followed
- [ ] Checked if private methods are properly declared and used
- [ ] Checked if hard-to-understand areas of code are commented
- [ ] Checked if tests are effective
- [ ] Built and deployed changes on dev VM and tested manually
- [x] (Checked if all type annotations were added and/or updated appropriately)
```

### CODE REVIEW

File: python/hsfs/engine/python.py [hunk 1]
```diff
@@ -121,12 +125,24 @@ def _sql_offline(
...
+                arrow_flight_client.get_instance().read_query,
+                sql_query,
+            )
         else:
             with self._create_hive_connection(
                 feature_store, hive_config=hive_config
             ) as hive_conn:
-                result_df = pd.read_sql(sql_query, hive_conn)
+                # Suppress SQLAlchemy pandas warning
+                with warnings.catch_warnings():
```
Comment (moritzmeister) [inline python/hsfs/engine/python.py:138]: Can we please not do this? I usually keep an eye on warnings for things that will be deprecated soon.

File: python/hsfs/util.py [hunk 2]
```diff
@@ -300,6 +303,38 @@ def translate_legacy_spark_type(output_type):
...
 
+def _loading_animation(message, stop_event):
+    for char in itertools.cycle([".", "..", "...", ""]):
+        if stop_event.is_set():
+            break
+        print(f"{message}{char}   ", end="\r")
+        time.sleep(0.5)
+
+
+def run_with_loading_animation(message, func, *args, **kwargs):
```
Comment (moritzmeister) [inline python/hsfs/util.py:314]: Can't we just use TQDM?

File: python/hsfs/engine/python.py (hunk 1 above)
Comment (tdoehmen) [reply python/hsfs/engine/python.py:138]: It's about this warning in particular: "UserWarning: pandas only support SQLAlchemy connectable(engine/connection) ordatabase string URI or sqlite3 DBAPI2 connectionother DBAPI2 objects are not tested, please consider using SQLAlchemy", we should get rid of it.

Deprecation warning ragarding the connection will still show up, like this one here, that's why I didn't include it in the block.
image

Comment (tdoehmen) [reply python/hsfs/engine/python.py:138]: we can specific the filter to only filter user warning `warnings.simplefilter("ignore", UserWarning),` then deprecation warning will still show. I'll make a commit.

**Pull request merged**